dev/sample_mqttplugin/__init__.py

```diff
@@ -88,6 +96,13 @@ def stop(self):
         self.logger.dbghigh(self.translate("Methode '{method}' aufgerufen", {'method': 'stop()'}))
         self.alive = False
 
+        # let the plugin change the state of pause_item
+        if self._pause_item:
+            self._pause_item(True, self.get_fullname())
+
+        # if you use schedulers, this stops all schedulers the plugin has started.
+        #self.scheduler_remove_all()
```

Comment:
I'd uncomment that cause there is no negative effect if the method is called even if no schedulers are created..?

Reply:
As the sample mqtt plugin has no schedulers and I didn't want to add "default complexity" to the sample plugin, I kept this as a comment.